apps/web/src/components/FeedItem/FeedItem.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import FeedItem from './FeedItem';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { ReactComponent as HomeIcon } from '../../assets/homeIcon.svg';
+
+export default {
+  title: 'Components/FeedItem',
+  component: FeedItem,
+} as ComponentMeta<typeof FeedItem>;
+
+const Template: ComponentStory<typeof FeedItem> = (args) => (
+  <FeedItem {...args} />
```

Comment:
Saw the big icon in the screen shot you posted to your PR description. Our `Icon` components will always grow to fill the box they are in. We can update how our story renders by modifying what is returned from out `Template`. Most valid JSX/TSX can be placed in the `return` statement of our `Template` function.

```suggestion
  <div style={{ height: "35px"}}>
    <FeedItem {...args} />
  </div>

```

Reply:
This will render the `FeedItem` component in a container with a fixed height, which should result in something closed to the designs.